Automattic/wp-calypso
# P2 signup: use .p2.blog domain

In this PR, we change the default domain for new P2 sites to `.p2.blog`.

## Testing instructions

Apply D44950-code to your sandbox and sandbox public-api. Navigate to `http://calypso.localhost:3000/start/p2` and go through the signup flow. You should end up with a new P2 site on a `.p2.blog` domain.
